src/common/LogEntry.cc

```diff
@@ -183,7 +183,8 @@ string clog_type_to_string(clog_type t)
       return "crit";
     default:
       ceph_abort();
-      return 0;
+      __builtin_unreachable();
+      return string{};
```

Comment:
i'd suggest just drop these two lines. `ceph_abort()` is marked with `[[noreturn]]`, and that'd suffice.

Reply:
Good point. I'll check and probably change the fix.

Reply:
fixed.